org.contourterminal.App.appdata.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <component type="desktop">
-  <id>org.contour_terminal.Terminal</id>
+  <id>org.contourterminal.App</id>
```

Comment:
NO,
The app-id was settled.

Reply:
To reiterate, it doesn't fit the requirements.

`.App` is just bad taste. 


`io.github.contour_terminal.contour` would be acceptable too.

Reply:
I don't like the underscore, so I am trying to get rid of it before it is too late. This was my whole point of my rather biggish post this morning. :)